specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -878,8 +883,9 @@ model ConversationSentimentResults extends PreBuiltResult{
 
 @added(Versions.v2023_04_15_preview)
 @doc("The result from sentiment analysis operation for each conversation.")
-model ConversationSentimentResultsConversation extends ConversationSentimentResult{
-  ...ConversationResultBase
+model ConversationSentimentResultsConversation{
```

Comment:
Nit:
```suggestion
model ConversationSentimentResultsConversation {
```

Reply:
I won't repeat, but some other instances below. Generally, we like to keep things "open" - makes it more reviewable.